e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js

```diff
@@ -24,7 +24,7 @@ function setLDAPTestSettings(config) {
 
 // assumes the CYPRESS_* variables are set
 // assumes that E20 license is uploaded
-// for setup with AWS: Follow the instructions mentioned in the mattermost/platform-private/config/ldap-test-setup.txt file
+// for setup with AWS: Follow the instructions mentioned in the mattermost/enterprise/config/ldap-test-setup.txt file
```

Comment:
another unrelated change but `platform-private` is no longer a thing

Reply:
you are right - I was mistaken for some reason the link failed to load for me and I assumed it was the wrong repo